src/reminders.py

```diff
@@ -0,0 +1,79 @@
+from telegram import Bot, Message, Update
+from telegram.ext import Job, JobQueue
+from typing import Tuple
+import logging
+
+from src import commands
+
+logger = logging.getLogger(__name__)
+
+REMINDER_MESSAGE = "I WILL POST PINNED MESSAGE HERE"
+REMINDER_PINNED_INTERVAL = 30 * 60
+REMINDER_SOCIAL_INTERVAL = 10 * 60
+REMINDER_SAFE_ON_THE_ROAD_INTERVAL = 120 * 60
+PINNED_JOB = "pinned"
+SOCIAL_JOB = "social"
+SAFE_ON_THE_ROAD_JOB = "safe_on_the_road"
+JOBS_NAME = [PINNED_JOB, SOCIAL_JOB, SAFE_ON_THE_ROAD_JOB]
+
+
+def send_social_reminder(bot: Bot, job: Job):
+    """send_reminder"""
+    chat_id = job.context
+    logger.info("Sending a social reminder to chat %s", chat_id)
+    results = commands.social_help()
+    bot.send_message(chat_id=chat_id, text=results, disable_web_page_preview=True)
+
+
+def send_pinned_reminder(bot: Bot, job: Job):
+    """send_reminder"""
+    chat_id = job.context
+    chat = bot.get_chat(chat_id)
+    msg: Message = chat.pinned_message
+    logger.info("Sending pinned message to chat %s", chat_id)
+
+    if msg:
+        bot.forward_message(chat_id, chat_id, msg.message_id)
+    else:
+        bot.send_message(chat_id=chat_id, text=REMINDER_MESSAGE)
+
+
+def send_safe_on_the_road_reminder(bot: Bot, job: Job):
+    chat_id = job.context
+    logger.info("Sending a safe on the road reminder to chat %s", chat_id)
+    bot.send_photo(chat_id, photo=open('resources/safe-on-the-road/ua.png', 'rb'))
```

Comment:
Why did you decide to add only 1 photo instead of all of them in different languages?

Also I would put the photo inside `src/`, only tests and infrastructure are normally outside. But probably doesn't matter for now because we will have to restructure the whole architecture anyway^^